docs/data-modeling/model.mdx

```diff
@@ -5,519 +5,208 @@ description: "Models allow you to define the structure of your data and how it c
-SELECT *
-FROM users
-WHERE name LIKE '%' || {{name}} || '%'
-  AND name > {{lower_bound}}
-  AND name < {{upper_bound}}
-```
-
-```sh title="Then, use the PostgreSQL connector's plugin to add the native query to your connector's configuration:"
-ddn connector plugin --connector <my_subgraph>/connector/<my_connector>/connector.yaml -- \
-  native-operation create --operation-path native_operations/queries/user_by_name_between.sql --kind query
-```
-
-```sh title="Introspect your PostgreSQL instance:"
-ddn connector introspect <connector_name>
-```
-
-```sh title="Then, update your models:"
-ddn model update <connector_name> "*"
-```
+    <ModelCreateNativeQueryHowToPostgreSQL />
 
   </TabItem>
   <TabItem value="MongoDB" label="MongoDB">
 
-Within your connector's directory, you can add a new JSON configuration file to define a
-[native query](/connectors/mongodb/native-operations/native-queries.mdx).
-
-```sh title="Create a new directory to store your native queries:"
-mkdir -p <my_subgraph>/connector/<connector_name>/native_queries/
-```
-
-```json title="Create a new file in your connector's directory:"
-// native_queries/user_by_name_between.json
-{
-  "name": "userByName",
-  "representation": "collection",
-  "description": "Look up a user by a regular expression match on name",
-
-  "inputCollection": "users",
-
-  "arguments": {
-    "name": { "type": { "scalar": "string" } },
-    "lower_bound": { "type": { "scalar": "string" } },
-    "upper_bound": { "type": { "scalar": "string" } }
-  },
-
-  "resultDocumentType": "UserByName",
-  "objectTypes": {
-    "UserByName": {
-      "fields": {
-        "_id": { "type": { "scalar": "objectId" } },
-        "name": { "type": { "scalar": "string" } }
-      }
-    }
-  },
-
-  "pipeline": [
-    {
-      "$match": {
-        "$and": [
-          { "name": { "$regex": "{{ name }}" } },
-          { "name": { "$gt": "{{ lower_bound }}" } },
-          { "name": { "$lt": "{{ upper_bound }}" } }
-        ]
-      }
-    }
-  ]
-}
-```
-
-```bash title="Use the DDN CLI to introspect your MongoDB instance:"
-ddn connector introspect <connector_name>
-```
-
-```sh title="Then, update your models:"
-ddn model update <connector_name> "*"
-```
+    <ModelCreateNativeQueryHowToMongoDB />
 
   </TabItem>
   <TabItem value="ClickHouse" label="ClickHouse">
 
-Within your connector's directory, you can add a new SQL configuration file to define a
-[native query](/connectors/clickhouse/native-queries.mdx).
+    <ModelCreateNativeQueryHowToClickHouse />
 
-```sh title="Create a new directory to store your native queries:"
-mkdir -p <my_subgraph>/connector/<connector_name>/queries/
-```
+  </TabItem>
+</Tabs>
 
-```sql title="Create a new file in your connector's directory:"
-// queries/UsersByName.sql
-SELECT *
-FROM "default"."users"
-WHERE "users"."name" = {name: String}
-```
+Find a tutorial on how to add a native query for your data connector in the relevant section [here](/).
```

Comment:
This can probably be dumped since we have the link for each connector in the tab above, right?